model/client4.go

```diff
@@ -3919,7 +3919,8 @@ func (c *Client4) GetRedirectLocation(urlParam, etag string) (string, *Response)
 	return MapFromJson(r.Body)["location"], BuildResponse(r)
 }
 
-func (c *Client4) RegisteTermsOfServiceAction(userId, termsOfServiceId string, accepted bool) (*bool, *Response) {
+// Saves action performed by a user against a specific terms of service.
```

Comment:
The method name should be prepended to the comment. This is the idiomatic way to write docs in go.

```suggestion
// RegisterTermsOfServiceAction saves an action performed by a user against a specific terms of service.
```

Reply:
@hanzei yes, of course. Might have been low of coffee :P Fixed everywhere.